incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+	"time"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+
+		for _, ns := range nses {
+			runSuccess("kubectl hns set", ns, "-a")
+			//remove subname annotation
+			runSuccess("kubectl annotate ns", ns, "hnc.x-k8s.io/subnamespaceOf-")
```

Comment:
Consider adding a `tryRun` function that doesn't insist on succeeding (e.g. in case the namespace doesn't exist). Maybe rename `runSuccess` to `mustRun`, and `mustRun` can call `tryRun`.

Reply:
Note that the call to `kubectl apply` should always succeed.

Reply:
done